requirements.txt

```diff
@@ -1,5 +1,5 @@
 pypif==2.0.1
-pypif_sdk==2.0.0
-citrination_client==2.1.0
+pypif_sdk>=2.2.1,<3
```

Comment:
These should be pinned in `requirements.txt` and open (like this) in setup.py (as they are).